components/atoms/icon/icon/__tests__/stories.js

```diff
@@ -26,13 +34,18 @@ export default {
     }
   }
 }
-
-export const main = () => {
-  const StylesObject = object('Style', {
-    color: '',
-    fontSize: ''
-  })
-
+const ListIcons = {
+  Copy: 'copy',
+  Copyright: 'copyright',
+  Images: 'images',
+  Smile: 'smile',
+  UserCircle: 'user-circle'
+}
+const IconPrefix = {
```

Comment:
@mhsnprvr3 This should be in the context not listed here - as we can reuse in other places